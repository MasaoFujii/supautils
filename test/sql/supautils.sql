-- build extension
create schema supa;
create extension supautils with schema supa;

select supa.get_one();
