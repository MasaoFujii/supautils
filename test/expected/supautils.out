-- build extension
create schema supa;
create extension supautils with schema supa;
select supa.get_one();
 get_one 
---------
       1
(1 row)

